db/migrate/20250827205335_migrate_cvv_message_to_redirect_auth_url.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class MigrateCvvMessageToRedirectAuthUrl < ActiveRecord::Migration[7.1]
+  def up
+    records = Spree::Payment.where.not(
+      cvv_response_message: nil
+    ).where.not(
+      state: :completed
+    )
```

Comment:
Can you combine the two conditions?
```suggestion
    records = Spree::Payment.where.not(
      cvv_response_message: nil,
      state: :completed
    )
```

Reply:
Thank you! I love learning new stuff. :smile: That makes sense.